main.js

```diff
@@ -1,45 +1,100 @@
-$(document).ready(function(){
-    $('.phone-number').mask('+7(000) 000-00-00');
-    });
+// testing
+    $(document).ready(function(){
+        $('.phone-number').mask('+7(000) 000-00-00');
+        });
+    
 
- let emailValid = /^([a-z0-9_\.-])+@[a-z0-9-]+\.([a-z]{2,4}\.)?[a-z]{2,4}$/;
-		$('#exampleInputEmail').keyup(function(){
-            !emailValid.test($('#exampleInputEmail').val())
-            ? $('#exampleInputEmail').css('border-color' , 'rgb(255, 0, 0)')
-            : $('#exampleInputEmail').css('border-color', '#007d7c');
-		});
+    let inputEmail = document.getElementById('inputEmail');
+    let pass = document.getElementById('inputPassword');
+    let passConfirm = document.getElementById('inputPasswordConfirm');
+    let showHidden = document.getElementById('showHidden');
+    let showHiddenConfirm = document.getElementById('showHiddenConfirm');
+    let inputNameValue, inputEmailValue, inputPhoneValue, inputPasswordValue;
+    let result = document.getElementById('result');
+    let formContent = document.getElementById('formContent');
+    
+    let setListener = (element, type, hendler) => {
+        if (!element){
+        return;
+    }
+    element.addEventListener(type, hendler);
+    return () => {
```

Comment:
You dont need this, function should not return another function, please remove this.